### PR TITLE
v2v: add case ‘Check minimum space and inodes for linux guest’

### PR DESCRIPTION
**Pull request by vwu-vera**
Please check TC #RHEL-254348 and bz #1972708 #1764569

Signed-off-by: vwu-vera <vwu@redhat.com>

### CODE REVIEW

File: v2v/tests/cfg/convert_from_file.cfg [hunk 1]
```diff
@@ -38,12 +38,24 @@
...
+                - null:
+                    only dest_null
     variants:
         - input_mode:
             variants:
                 - disk:
                     only source_none
                     input_mode = "disk"
+                    only output_mode.null
+                    disk_dir = 'DISK_DIR_V2V_EXAPLE'
```
Comment (kamilvarga) [inline v2v/tests/cfg/convert_from_file.cfg:50]: There seems to be a typo in the EXAMPLE word.

File: v2v/tests/cfg/convert_from_file.cfg [hunk 2]
```diff
@@ -38,12 +38,24 @@
...
+                    variants:
+                        - less_inodes:
+                            only negative_test
+                            input_file = '${disk_dir}/DISK_LESS_INODES_V2V_EXAMPLE'
```
Comment (kamilvarga) [inline v2v/tests/cfg/convert_from_file.cfg:54]: Just out of curiosity, are those files existing somewhere in the repo? I have no experience with these V2V tests, so I am wondering...

**Review by kamilvarga — COMMENTED** [review]
Others LGTM.

File: v2v/tests/cfg/convert_from_file.cfg (hunk 1 above)
Comment (vwu-vera) [reply v2v/tests/cfg/convert_from_file.cfg:50]: Thanks Kamil.

File: v2v/tests/cfg/convert_from_file.cfg (hunk 2 above)
Comment (vwu-vera) [reply v2v/tests/cfg/convert_from_file.cfg:54]: Yes. We put these prepared files on NFS server and mount it on conversion server. So we can use it directly for testing.

**Comment by xiaodwan** [comment]
I suggest you use a prepared guest on esx server. Those codes are hard to maintain and easy going fault.

File: v2v/tests/cfg/convert_from_file.cfg (hunk 2 above)
Comment (kamilvarga) [reply v2v/tests/cfg/convert_from_file.cfg:54]: Thank you for your answer. One more question. Are those files backup and versioned somewhere as well? So we can rerun test anytime in the future on any machine.

**Comment by vwu-vera** [comment]
(1/1) type_specific.io-github-autotest-libvirt.function_test_esx.positive_test.inodes.more_inodes.esx_70.rhev.rhv.it_default: PASS (4420.06 s)

 (1/1) type_specific.io-github-autotest-libvirt.function_test_esx.positive_test.inodes.more_inodes.esx_70.rhev.rhv.it_default: PASS (4671.98 s)

**Comment by vwu-vera** [comment]
@xiaodwan Changed the source from file to ESX VMs. Please help to check the commit. Thanks.

**Review by xiaodwan — APPROVED** [review]

**Pull request merged**